main.cpp

```diff
@@ -1,22 +1,13 @@
-#include "clang/Frontend/FrontendActions.h"
-#include "clang/Tooling/CommonOptionsParser.h"
-#include "clang/Tooling/Tooling.h"
-#include "llvm/Support/CommandLine.h"
+#include <iostream>
 
-using llvm::cl::OptionCategory;
-using clang::tooling::CommonOptionsParser, \
-      clang::tooling::ClangTool, \
-      clang::tooling::newFrontendActionFactory;
-using clang::SyntaxOnlyAction;
+#include "include/Utils.h"
 
-static OptionCategory TransformationCategory("Basic transformation");
+using std::cerr, std::endl;
 
 int main(int argc, const char **argv) {
```

Comment:
This function should do the next steps:
1. Check that there are at least 2 arguments in `argv` (the first program name, the second path to config)
2. Try to parse config (catch an exception if something went wrong and print the corresponding message)
3. Initialize `Runner` with vector of transformations.
4. For all args after argv[1] apply `Runner` (call `run`).

Reply:
Also, there are some hyperparameters in config (output directory, the number of augmentations per file). We should extract it too and use it in Runner.